### DISCUSSION TITLE
Missing Event Handling for Pill Removal in MultiSelect Component

### DISCUSSION THREAD

**Discussion by davlatsultonov** [mantinedev/mantine#5775] [Feature requests]
### Dependencies check up

- [x] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.3.2

### What package has an issue?

@mantine/core

### What framework do you use?

Vite

### In which browsers you can reproduce the issue?

All

### Describe the bug

I'm having trouble getting the ID of a deleted item (pill) from the user interface to use it in an API request. Right now, the "onChange" handler only gives back an array of values, so I can't tell which item was removed. As a result, I can't grab the ID of the deleted pill for further API actions. Any ideas on how to solve this? Thanks!

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
I'll check whether this feature can be added in one of the next minor releases

**Comment by davlatsultonov** [comment]
The requested feature was added as `onRemove` function handler in `multiselect` component.
